platform/ios/ios.xcodeproj/project.pbxproj

```diff
@@ -511,8 +511,6 @@
 		DA8963381CC549A100684375 /* sprites in Resources */ = {isa = PBXBuildFile; fileRef = DA8963341CC549A100684375 /* sprites */; };
 		DA8963391CC549A100684375 /* styles in Resources */ = {isa = PBXBuildFile; fileRef = DA8963351CC549A100684375 /* styles */; };
 		DA89633A1CC549A100684375 /* tiles in Resources */ = {isa = PBXBuildFile; fileRef = DA8963361CC549A100684375 /* tiles */; };
-		DA9EA82B201C0C0C00F9874D /* NSExpression+MGLAdditions.h in Headers */ = {isa = PBXBuildFile; fileRef = DA9EA82A201C0C0B00F9874D /* NSExpression+MGLAdditions.h */; settings = {ATTRIBUTES = (Public, ); }; };
```

Comment:
This header is intentionally public so that clients like the React Native SDK can use familiar JSON-like objects instead of composing NSExpression format strings.

/cc @nitaliano

Reply:
This was not intentional.

Reply:
Can a test be added that ensures this will remain the case?

Reply:
I was looking at the project and this header is marked as public. Not sure what happened.

Reply:
@1ec5 looks like this header was duplicated and somehow this commit removed the duplicated entries. Although it's still public and I didn't do anything to remove it.

Reply:
OK, as long as the header is still part of both the dynamic and static targets, we’re in good shape. Xcode sometimes grooms the project file to remove duplicate references that may sneak in due to bad merges.